orte/tools/orterun/orterun.1in

```diff
@@ -13,10 +13,12 @@
 .SH NAME
 .
 orterun, mpirun, mpiexec \- Execute serial and parallel jobs in Open MPI.
+oshrun, shmemrun \- Execute serial and parallel jobs in Open SHMEM.
```

Comment:
Er... this is the orterun man page.  You can't just change the name to the OSHMEM executable names here.  (ditto for later in the file).

Perhaps you edited this through a sym link and didn't realize that it was the actual orterun man page, and not a page unique to OSHMEM?